src/sourceItemWrapper.js

```diff
@@ -489,20 +489,92 @@ class SourceItemWrapper extends ItemWrapper {
         // offer to automatically link to zotero items
     }
 
-    getFromBibTeX() {
-        Services.prompt.alert(
-            window,
-            Wikicite.getString('wikicite.global.unsupported'),
-            Wikicite.getString('wikicite.bibtex.import-citations.unsupported')
+    // get BibTeX from clipboard
+    // - also supports other formats supported by Zotero
+    // - also supports multiple items
+    async getFromBibTeX() {
+        var str = Zotero.Utilities.Internal.getClipboard("text/unicode");
+        var translation = new Zotero.Translate.Import();
```

Comment:
Zotero's translation system may not be ready. Please add `await Zotero.Schema.schemaUpdatePromise;` above this line.